File: ros_ws/src/my_robot_tutorials/src/my_first_node.cpp
#include "ros/ros.h"

int main(int argc, char **argv)
{
    ros::init(argc, argv, "my_first_cpp_node");
    ros::NodeHandle nh;  
    ROS_INFO("Node has been created");

    ros::Rate rate(10);
    
    while(ros::ok()) {
    	ROS_INFO("Standing here...");
	rate.sleep();
    }

    return 0;
}

File: ros_ws/src/my_robot_tutorials/scripts/my_first_node.py
#!/usr/bin/env python

import rospy


def main(args=None):
    rospy.init_node("my_first_python_node")
    rospy.loginfo("Hello there, this node has been started")

    rate = rospy.Rate(10)

    while not rospy.is_shutdown():
        rospy.loginfo("Standing here...")
        rate.sleep()


if __name__ == "__main__":
    main()    